package.json

```diff
@@ -14,8 +14,9 @@
   },
   "devDependencies": {
     "eslint": "^1.10.3",
+    "jasmine-ajax": "^3.2.0",
     "jasmine-core": "^2.4.1",
-    "karma": "^0.13.15",
+    "karma": "=0.13.15",
```

Comment:
I experienced a Karma bug on version `0.13.16` (https://github.com/karma-runner/karma/issues/1768) so locking to the previous patch for now.